src/CreeDictionary/CreeDictionary/paradigm/panes.py

```diff
@@ -350,14 +355,17 @@ def inflection_cells(self) -> Iterable[InflectionTemplate]:
         # a header, by definition, has no inflections.
         return ()
 
+    @property
+    def fst_tags(self) -> tuple[str, ...]:
+        return self._tags
+
     def __eq__(self, other) -> bool:
         if isinstance(other, HeaderRow):
             return self._tags == other._tags
         return False
 
     def __str__(self):
-        tags = "+".join(self._tags)
-        return f"{self.prefix} {tags}"
+        return " ".join(f"{self.prefix} {tag}" for tag in self._tags)
```

Comment:
`self.prefix` will be repeated for every tag?

Reply:
Yes... this is the syntax I came up with and did not fully think out 🙃 

This syntax is consistent with row labels and column labels. It's still not too late to change the syntax if you think it's silly.